package.json

```diff
@@ -49,7 +49,7 @@
   },
   "dependencies": {
     "@solidity-parser/parser": "0.14.5",
-    "@syntest/framework": "^0.1.2",
+    "@syntest/core": "^0.2.0",
```

Comment:
We should probably lock down the version as the current syntax would propagate all changes

Reply:
Fixed